home/home_2.1/main.swift

```diff
@@ -0,0 +1,92 @@
+import Foundation
+
+let FILENAME = "user_stats.txt"
+
+func readStats() -> [String: Int] {
```

Comment:
we can try to do some encapsulation using structures as described for c++ version.